### DISCUSSION TITLE
Problems of developing a new model

### DISCUSSION THREAD

**Discussion by cqss11** [CURENT/andes#457] [Data Format]
Hi, hantao!
I am developing a new model. Howerver,

-> Single process finished in 1.1314 seconds.
GENROU (vf range) out of typical lower limit.

idx | values | limit
----+--------+------
 2  | 0.991  | 1

Suspect initialization issue! Simulation may crash!

    Name      | Var. Value | Eqn. Mismatch
--------------+------------+--------------
 a Bus 1      | 0.000      | 0.149
 a Bus 8      | 0.007      | -0.069
 a Bus 10     | -0.059     | -0.055
 a Bus 13     | -0.091     | -0.025
 v Bus 1      | 1.000      | 0.010
 v Bus 8      | 1.000      | -0.021
 v Bus 10     | 1.000      | 0.160
 v Bus 13     | 1.000      | -0.150
 LL_y TGOV1 1 | 0.149      | 0.527
 LL_y TGOV1 2 | 0.080      | 0.672
 LL_y TGOV1 3 | 0.025      | 0.788

Initialization failed!!

Then how do I debug parameters？looking forward to your advice!

**Comment by cuihantao** [comment]
You probably provided wrong `bus` for the dynamic generators. The `bus` field should be the indices of buses to which `gen` are connected.
